### ISSUE TITLE
Lens Correction Module Does Not Identify Camera, Nor Will Allow for Manual Corrections

### ISSUE THREAD

**Issue by SprinterChair** [darktable-org/darktable#8203]
Describing the Bug -

With no image, even those where the lens correction module was previously turned on and working properly, is the camera identified (Canon Rebel T7). The module prompts me to make choose camera or make adjustments manually. 

Clicking on the combo box for cameras or the arrow for the drop down menu to the right of the box does nothing. 

All options for manual corrections within the module are greyed out, making them unusable as well. 

To Reproduce -

1. Open image taken on Canon Rebel T7 in darkroom.
2. Open and turn on lens correction module.
3. Turning on the module will fail to make any changes to the image. 
4.  Attempt to change any settings in the module will fail. 

Expected Behavior -

Lens correction module will automatically identify camera model as it has in previous versions.
Lens correction module will automatically apply corrections when turned on. 

Screenshots -

![image](https://user-images.githubusercontent.com/62212860/107839911-f80e6f80-6d7c-11eb-87d7-f483d9fc691e.png)
^^^This shows the module when first opened. Notice how the camera isn't identified. Arrow which should produce a drop down menu for camera selection does nothing. 

![image](https://user-images.githubusercontent.com/62212860/107839972-2be99500-6d7d-11eb-8b1c-7de37c148f71.png)
^^^This shows no change in the image when the module is turned on.

![image](https://user-images.githubusercontent.com/62212860/107840001-5b989d00-6d7d-11eb-86b6-5fc2c3f6f929.png)
^^^Interestingly, when right clicking on the combo box, a small black box appears next to the cursor. 

Platform- 

darktable version 3.4.1.1
Windows 10
OpenCL not activated (to my knowledge?)
Unsure of graphics card or drivers. (Can likely be provided if required. I would just have to do a bit of research on my laptop)

Additional context -

I would attach below two of my raw files for examples, but GitHub is telling me .CR2 and .xmp are not supported. If there were a different way for me to share them please let me know. 

Edit - 
Going to darktable > lib > libgphoto2 shows that the latest version of the gPhoto2 software, 2.5.26, was installed with the 3.4.1 update of dartable. Will be looking into whether the LensFun database is up to date as well (which I think I need some help with doing...)

**Comment by ptilopteri** [comment]
https://www.darktable.org/resources/camera-support/
search for "Canon EOS Rebel T7"

**Comment by SprinterChair** [comment]
I checked your link and do see that the Rebel T7 is lacking support, but that doesn't explain why I can't make manual adjustments or select a similar camera from the drop down menu. 

Also, if my camera isn't supported, how was I using the module just fine a few months ago...

**Comment by ptilopteri** [comment]
if you bothered to read more than just the lack of support, you will find how
to get your camera supported, but will take some effort on your part.

**Comment by SprinterChair** [comment]
I did read about getting camera support and did see that it would take some effort. I will consider doing this at some point. I made my previous comment stating that regardless of support, I am unable to make manual corrections with the module. I am going to close this thread because I will be looking into getting support, but that does not change the issue that the module should still allow for manual adjustments when camera support isn't available. 

I understand that you get many people on here that complain about something not working properly and do little to fix it. I'm trying my hardest to be helpful even though I have very little technical knowledge of coding/programming. I appreciated your help by directing me to a useful link but you didn't have to be rude. As I said, I with what little technical knowledge I have I will be looking into how I can help get camera support.
